sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/rule/CompositeParam.java

```diff
@@ -0,0 +1,104 @@
+package com.alibaba.csp.sentinel.adapter.gateway.common.rule;
```

Comment:
Could you please add the license header?

Reply:
fixed